test/unit/autocomplete.ts

```diff
@@ -215,6 +215,19 @@ suite('Autocomplete', ({ expect, spy }) => {
       expect(set).to.be.calledWith({ suggestions, navigations, categoryValues, selected: -1 });
       expect(setActivation).to.be.calledWith(targets, selected, false);
     });
+
+    it('should inactivate sayt when there are no suggestions', () => {
+      const suggestions = [];
```

Comment:
why do we need references to these objects if they aren't used in the expectations?